internal/clientio/resp.go

```diff
@@ -205,6 +205,26 @@ func Encode(value interface{}, isSimple bool) []byte {
 		return []byte(fmt.Sprintf("*%d\r\n%s", len(v), buf.Bytes()))
 	case map[string]bool:
 		return RespNIL
+	case []DiceCmdMeta:
```

Comment:
@aashraybhandar1 can you please rebase once and validate, this call would error out as `DiceCmdMeta` is part of `eval` now.
Also I believe we should keep type as `[]interface{}` we have similar response in `QWATCH` you can take a reference of `CreatePushResponse` function may be.